plugins/rtauto/plugin_base.dune

```diff
@@ -0,0 +1,6 @@
+(library
+ (name rtauto_plugin)
+ (public_name coq.plugins.rtauto)
+ (synopsis "Coq Plugin for RTAUTO")
+ (flags (-rectypes))
```

Comment:
There is something fishy here. All this code depends on coq. Coq is compiled with -rectypes and -rectypes is, AFAIK, viral. Why do you need to repeat it over and over? (Eg in findlib is the library using rectypes that declares it, not its users).

Reply:
Umm, IIRC my projects using findlib and `META.coq` needed to add `-rectypes` manually; maybe I did something wrong. It is easy to try and see.

Anyways, Dune now supports a per-project `flags` declaration, so these repetition will go away.

Reply:
If dune supports this option, why don't you use it? What do you mean by "will go away"?

Reply:
> If dune supports this option, why don't you use it?

As noted in the original PR TODO list this was pending on the release of Dune 1.0 so will be implemented soon and before merging.